plugins/msg-broker/mcollective/config/initializers/openshift-origin-msg-broker-mcollective.rb

```diff
@@ -39,7 +39,8 @@
       :node_profile_enabled => conf.get_bool("NODE_PROFILE_ENABLED", "false"),
       :node_rsync_address => conf.get("NODE_RSYNC_ADDRESS", "ip_address"),
       :quota_blocks_buffer => conf.get("QUOTA_BLOCKS_BUFFER", 1.2),
-      :quota_inodes_buffer => conf.get("QUOTA_INODES_BUFFER", 1.2)
+      :quota_inodes_buffer => conf.get("QUOTA_INODES_BUFFER", 1.2),
+      :node_df_buffer => conf.get("NODE_DF_BUFFER", 0.05)
```

Comment:
I don't think we need this since we are getting this as a fact about the node, so it should be coming from `resource_limits.conf`, not the broker-side configuration.